onnx/defs/tensor/defs.cc

```diff
@@ -1417,4 +1417,26 @@ ONNX_OPERATOR_SET_SCHEMA(
                                      }}
       ));
 
+static const char* NonZero_ver9_doc = R"DOC(
+    Returns the indices of the elements that are non-zero
+    (in row-major order - by dimension).
+    NonZero behaves similar to numpy.nonzero:
+    https://docs.scipy.org/doc/numpy/reference/generated/numpy.nonzero.html
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    NonZero,
+    9,
+    OpSchema()
+    .SetDoc(NonZero_ver9_doc)
+    .Input(0, "X", "input", "T")
+    .Output(0, "Y", "output", "tensor(int64)")
```

Comment:
Let's make it clear here the output is always a 2d tensor.

Reply:
done